### DISCUSSION TITLE
migrate from csv-importer to data-importer

### DISCUSSION THREAD

**Discussion by TWART016** [firefly-iii/firefly-iii#6128] [General]
The csv-importer image does not exist anymore. Therefore I have to change from csv-importer to data-importer. Is it possible to just change the image inside the docker-compose file?

In the end I import the csv with a script
`docker exec -t fireflyiii_import php artisan importer:import <folder> /import/configurations/file.json`

**Comment by JC5** [comment]
You have to reverse the arguments (file.json before csv.csv). Otherwise, it should all be the same.

**Reply by TWART016** [reply]
I changed the image. While opening the GUI I get the error:
`You can't set FIREFLY_III_ACCESS_TOKEN together with FIREFLY_III_CLIENT_ID. One must remain empty.`

Both is still set in the compose file.
